test/functionalTest/harnessFunctions.sh

```diff
@@ -595,22 +625,200 @@ function dbInsertEntity()
 #   --host        <host>           (default: localhost)
 #   --port        <port>           (default: $CB_PORT)
 #   --url         <URL>            (default: empty string)
+#   --urlParams   <params>         (URI parameters 'in' URL-string)
```

Comment:
I have realized that it is not a  new parameter (it was in the previous version of the function), but no .test is using, anyway...